### PR TITLE
GTFS import: mount local download.sh into postgtfs-gtfs-importer

### PR DESCRIPTION
**Pull request by derhuerst**
This is the `ipl-dagster-pipeline` equivalent of https://github.com/mobidata-bw/ipl-orchestration/pull/265.

related: https://github.com/mobidata-bw/postgis-gtfs-importer/issues/10

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The changelog lists most feature changes between each release. 
 
+## 2024-10-29
+
+- GTFS import: Mount local (modified) `download.sh` into `postgis-gtfs-importer` container.
+	- ⚠️ `$IPL_GTFS_IMPORTER_HOST_CUSTOM_SCRIPTS_DIR` must contain a `download.sh`, so if it doesn't, this is a breaking change!
```
Comment (the-infinity) [inline CHANGELOG.md:8]: Why suddenly tabs?

File: pipeline/assets/gtfs.py [hunk 2]
```diff
@@ -41,6 +41,7 @@
             'volumes': [
                 os.path.join(os.getenv('IPL_GTFS_IMPORTER_HOST_GTFS_OUTPUT_DIR'), ':/var/gtfs/:rw'),
```
Comment (the-infinity) [inline pipeline/assets/gtfs.py:42]: Would remove the `os.path.join`, too, as this is not a path join.

File: CHANGELOG.md (hunk 1 above)
Comment (derhuerst) [reply CHANGELOG.md:8]: I don't know. 🤷 – Is it sudden? The `.editorconfig` doesn't define any rules for `.md` files.

Shall we change to 4 spaces?

Comment (the-infinity) [reply CHANGELOG.md:8]: Yea, at least every file I know before is with spaces.

**Review by the-infinity — APPROVED** [review]

**Pull request merged**